Events/services.js

```diff
@@ -0,0 +1,44 @@
+// This function return a json containing the eventuid of the newly created event.
+var createEvent = function(req) {
+  console.log(req);
+
+  // TODO (zachlefevre): Write method and remove console.log().
+  return req; // This will be replaced with a json containing the eventuid of the newly created event.
```

Comment:
Send back a fake object (status and expected object) that front-end team can use.

In this case, the object would look something like:
```javascript
return { 
  status: 200,
  object: {
    "category": "advisor meeting",
    "contact": "Advisor Name",
    "Creator": 0,
    "datetime": "Tue Feb 28 2017", // This may not be the correct format
    "description": "I spoke with my advisor. It was cool.",
    "location": "EC 204",
    "title": "Hello, advisor!"
  }
};
```

That way we can continue developing the back-end without leaving the front-end guys out to dry.